edx_exams/apps/core/api.py

```diff
@@ -0,0 +1,80 @@
+"""
+Library for the edx-exams service.
+"""
+import logging
+from datetime import datetime
+
+import pytz
+
+from edx_exams.apps.core.models import ExamAttempt
+from edx_exams.apps.core.statuses import ExamAttemptStatus
+
+log = logging.getLogger(__name__)
+
+
+def update_attempt_status(attempt_id, to_status):
```

Comment:
this is an extremely paired down version of what exists in edx-proctoring. I would like to keep it this way if possible. Any additional logic that needs to be added for grading, credentials, completion, and name verification should be added outside of this function so we can limit the scope of this to only updating the attempt status. This function should make the calls to trigger those behaviors, rather than define those behaviors.

Reply:
1000% agreed. Maybe we even use signals/celery to further decouple these behaviors from the UI request thread? Or event bus if that's a viable option. Later problems either way